packages/mui-varnish/src/util.ts

```diff
@@ -0,0 +1,15 @@
+export type Dictionary<T> = { [id: string]: T };
+
+export type Indexable<T> = { [K in keyof T & string]: T[K] };
+
+export function pxToRem(px: number, base: number = 16) {
+  return `${px / base}rem`;
+}
+
+export function dictionaryToArray<T>(dict: Dictionary<T>) {
+  return Object.keys(dict).map((key) => dict[key]);
+}
+
+export function strIncludes(filter: string, value: string) {
+  return value.toLowerCase().includes(filter.toLowerCase());
+}
```

Comment:
The name, IMO, should emphasize that it's case insensitive.

FWIW this is one of those places where I think a little bit of duplication is better than the abstraction. I think there's some loss of clarity by consolidating this. 

In other words I'd consider not generalizing this and just repeating the code you have here elsewhere too.

Oh, and you might consider using `localeCompare`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
removed